Source/cmake/OptionsGTK.cmake

```diff
@@ -472,6 +472,17 @@ if (USE_OPENJPEG)
     endif ()
 endif ()
 
+include(WOFF2Checks)
+if (FREETYPE_WOFF2_SUPPORT_IS_AVAILABLE)
+    SET_AND_EXPOSE_TO_BUILD(HAVE_WOFF_SUPPORT ON)
```

Comment:
I think this should be `HAVE_FREETYPE_WOFF2_SUPPORT`, because at first I understood the flag as “we can have WOFF2 support” but it actually means “we can have WOFF2 support *from Freetype*”.

Other than that, I like it that the patch is simple and mostly about determining whether Freetype has the needed support. That would allow us to keep the fallback to `woff2dec` while supported distributions don't have the needed Freetype version.

I am making a local build with this applied to test it a bit. Thanks for the patch!

@clopez, @carlosgcampos: WDYT about this?

Reply:
I used that since CoreText has native support for WOFF(2). Open to different names but the idea was it was also applicable to Cocoa platforms.